dotnet/src/webdriver/Remote/HttpCommandExecutor.cs

```diff
@@ -286,13 +286,25 @@ private async Task<HttpResponseInfo> MakeHttpRequest(HttpRequestInfo requestInfo
                 if (_logger.IsEnabled(LogEventLevel.Trace))
                 {
                     _logger.Trace($">> {requestMessage}");
+
+                    if (requestMessage.Content != null)
+                    {
+                        string content = await requestMessage.Content.ReadAsStringAsync();
+                        _logger.Trace($">> Body: {content}");
+                    }
                 }
 
                 using (HttpResponseMessage responseMessage = await this.client.SendAsync(requestMessage).ConfigureAwait(false))
                 {
                     if (_logger.IsEnabled(LogEventLevel.Trace))
                     {
                         _logger.Trace($"<< {responseMessage}");
+
+                        if ((int)responseMessage.StatusCode < 200 || (int)responseMessage.StatusCode > 399)
+                        {
+                            string responseBody = await responseMessage.Content.ReadAsStringAsync();
```

Comment:
Please don't read from response to log something. I believe in execution flow later somebody reads it, I hope it's time to log it as already read body.

Reply:
Removed the logger logic and added it to `HttpClientHandler `instead